spec/rester/client_spec.rb

```diff
@@ -178,7 +164,18 @@ def json_h(params)
 
     describe '#request', :request do
       let(:adapter) { double('adapter') }
+      let(:valid_response) {
+        [200, { 'http_x_rester_producer_name' => ["DummyService"]}, '']
```

Comment:
Is this actually a valid response?  I don't think the header should be in the http format.